general/administration/installing.md

```diff
@@ -722,3 +723,55 @@ Raw Ubuntu packages, including old versions, are available [here](https://jellyf
     sudo systemctl restart jellyfin
     sudo /etc/init.d/jellyfin stop
     ```
+
+#### Migrating native Debuntu install to docker
+
+It's possible to map your local installation's files to the official docker image.
+
+> [!Note]
+> You need to have exactly matching paths for your files inside the docker container! This means that if your media is stored at `/media/raid/` this path needs to be accessible at `/media/raid/` inside the docker container too - the configurations below do include examples.
+
+To guarantee proper permissions, get the `uid` and `gid` of your local jellyfin user and jellyfin group by running the following command:
+
+   ```sh
+      id jellyfin
+   ```
+
+You  need to replace the `<uid>:<gid>` placeholder below with the correct values.
+
+**Using docker**
+
+   ```sh
+   docker run -d \
+       --user <uid>:<gid> \
+       -e JELLYFIN_DATA_DIR=/var/lib/jellyfin \
+       -e JELLYFIN_CONFIG_DIR=/etc/jellyfin \
+       -e JELLYFIN_CACHE_DIR=/cache \
+       --volume </path/to/media>:</path/to/media> \
+       --net=host \
+       --restart=unless-stopped \
+       jellyfin/jellyfin
+   ```
+
+**Using docker-compose**
+
+   ```yml
+   version: "2"
+   services:
+   jellyfin:
+       image: bitwrk/jellyfin-rffmpeg:unstable
```

Comment:
You used a different docker image here. Probably best to use the official one in both places.